src/os/CMakeLists.txt

```diff
@@ -141,6 +137,32 @@ if(WITH_SPDK)
   endforeach()
 endif(WITH_SPDK)
 
+if(WITH_PMEM)
+  include(ExternalProject)
+  ExternalProject_Add(nvml_ext
+    DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
+    GIT_REPOSITORY "https://github.com/ceph/nvml.git"
+    GIT_TAG  "dd5b62dbc2cbbe048087dea"
+    SOURCE_DIR ${CMAKE_BINARY_DIR}/src/nvml
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND $(MAKE)
+    BUILD_IN_SOURCE 1
+    INSTALL_COMMAND "true")
+
+  ExternalProject_Add_Step(nvml_ext forcebuild
+    DEPENDEES configure
+    DEPENDERS build
+    COMMAND "true"
+    ALWAYS 1)
+
+  add_library(pmem STATIC IMPORTED GLOBAL)
+  add_dependencies(pmem nvml_ext)
+  target_link_libraries(os  pmem)
```

Comment:
remove this extra space, and move this line down to after `set_target_properties(pmem ...)`.

Reply:
> remove this extra space, 

@majianpeng not addressed